ArchiSteamFarm.CustomPlugins.ExamplePlugin/CatAPI.cs

```diff
@@ -30,15 +31,15 @@ namespace ArchiSteamFarm.CustomPlugins.ExamplePlugin;
 // You've always wanted from your ASF to post cats, right? Now is your chance!
 // P.S. The code is almost 1:1 copy from the one I use in ArchiBot, you can thank me later
 internal static class CatAPI {
-	private const string URL = "https://aws.random.cat";
+	private const string URL = "https://api.thecatapi.com";
 
 	internal static async Task<Uri?> GetRandomCatURL(WebBrowser webBrowser) {
 		ArgumentNullException.ThrowIfNull(webBrowser);
 
-		Uri request = new($"{URL}/meow");
+		Uri request = new($"{URL}/v1/images/search");
 
-		ObjectResponse<MeowResponse>? response = await webBrowser.UrlGetToJsonObject<MeowResponse>(request).ConfigureAwait(false);
+		ObjectResponse<List<MeowResponse>>? response = await webBrowser.UrlGetToJsonObject<List<MeowResponse>>(request).ConfigureAwait(false);
```

Comment:
Can we make this a read-only collection?

Does it have to be a list or can it be a set?

Reply:
I guess.
No idea. Should make no difference in this case, since we will always get only one element. If we decide to get more images at once in the future... well, I don't know if they can be duplicated.